tasks/check-i18n.js

```diff
@@ -32,25 +38,44 @@ async function run () {
 
   const usedKeysByFile = await getUsedKeys(sourceFilepaths);
 
+  console.log(
+    chalk
+      .bgHex('#EAEAEA')
+      .hex('#333333')
+      .bold(`\n ⌛ Checking translations for ${sourceFilepaths.length} files... `),
+  );
+
   // MISSING KEYS
   sourceFilepaths.forEach((src) => {
-    console.log(`Inspecting translations for ${src}`);
-
     Object.entries(translationsByLang).forEach(([lang, translations]) => {
 
       const missingKeys = usedKeysByFile[src]
         .filter((key) => translations[key] == null);
 
       if (missingKeys.length !== 0) {
         errors = true;
-        const formattedLog = missingKeys
-          .map((k) => `  MISSING (${lang}): ${k}`)
-          .join('\n');
-        console.log(formattedLog);
+
+        missingKeys.forEach((key) => {
+          if (!allMissingKeys.has(key)) {
+            allMissingKeys.set(key, new Set());
+          }
+          allMissingKeys.set(key, allMissingKeys.get(key).add(lang));
+        });
       }
     });
   });
 
+  const hasMissingKeys = allMissingKeys.size !== 0;
+  if (hasMissingKeys) {
+    console.log(chalk.bgHex('#be242d')(`\n ${allMissingKeys.size} keys are missing `));
+
+    const formattedMissingKeys = [];
+    allMissingKeys.forEach((values, key) => {
+      formattedMissingKeys.push(['> ' + key, [...values].join(',')]);
+    });
+    console.log(table(formattedMissingKeys));
+  }
+
   // UNUSED KEYS
   const allUsedKeys = Object.values(usedKeysByFile)
     .reduce((a, b) => [...a, ...b]);
```

Comment:
Hey I know this is not in the scope of the quick win cause this code was already here but I feel like this `reduce` is confusing, it's actually just a `flat()` no ?